src/nvim/ex_cmds.c

```diff
@@ -5324,13 +5324,11 @@ static void helptags_one(char_u *const dir, const char_u *const ext,
 
   FreeWild(filecount, files);
 
-  if (!got_int) {
+  if (!got_int && ga.ga_data != NULL) {
     /*
      * Sort the tags.
      */
-    if (ga.ga_data != NULL) {
```

Comment:
This suggests that execution of the function continues when `ga_data ==NULL`

Line 5339: for clang here is potential null pointer dereference, as connection between `ga_len` and `ga_data` is implicit